README.md

```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
 The command removes all the Kubernetes components associated with the chart and
 deletes the release.
 
-## Advanced Configuration
+### Advanced Configuration
 
 To fully configure the Helm chart, see the [advanced
 configuration](docs/advanced-configuration.md).
 
+## Setup Splunk
+
+Splunk OpenTelemetry Collector for Kubernetes supports installation using Helm. Read the Prerequisites, Installation and Deployment documentation before you start your deployment of Splunk OpenTelemetry Collector for Kubernetes.
+
+Perform the following steps before you install:
+
+1. Create a minimum of one Splunk platform index. This index events index, which will handle logs.
+If you do not configure this index, Splunk OpenTelemetry Collector for Kubernetes uses the defaults created in your HTTP Event Collector (HEC) token.
+
+2. Create a HEC token if you do not already have one. If you are installing the connector on Splunk Cloud, file a ticket with Splunk Customer Service and they will deploy the indexes for your environment, and generate your HEC token.
+
+### Setup for Non-Root User Group
+
+It is best practice to run pods as a non-root user. To avoid running collector pod as `root` user, perform below steps on each kubernetes nodes.
+
+In this chart, it is set to run as as a user with UID and GID of `10001` ([set here](https://github.com/splunk/sck-otel/blob/main/charts/sck-otel/values.yaml#L104)). But this user does not have the permission to read container log files typically owned by `root`. Below steps create a user with GID 10001 and grant access to that GID.
+
+```bash
+## create a user otel with uid=10001 and gid=10001
+sudo adduser --disabled-password --uid 10001 --no-create-home otel
+
+## setup a directory for storing checkpoints
+sudo mkdir /var/lib/otel_pos
+sudo chgrp otel /var/lib/otel_pos
+sudo chmod g+rwx /var/lib/otel_pos
+
+## setup container log directories.
+## To check where the files are, check symlinks file on `/var/log/pods/` and its target paths.
+ls -Rl /var/log/pods
+## default paths are these
+## `/var/lib/docker/containers` for docker
+## `/var/log/crio/pods` for cri-o
+## `/var/log/pods` for containerd
+## add your container log path if different
+if [ -d "/var/lib/docker/containers" ]
+then
+    sudo chgrp -R otel /var/lib/docker/containers
+    sudo chmod -R g+rwx /var/lib/docker/containers
+    sudo setfacl -Rm d:g:otel:rwx,g:otel:rwx /var/lib/docker/containers
+fi
+
+if [ -d "/var/log/crio/pods" ]
+then
+    sudo chgrp -R otel /var/log/crio/pods
+    sudo chmod -R g+rwx /var/log/crio/pods
+    sudo setfacl -Rm d:g:otel:rwx,g:otel:rwx /var/log/crio/pods
+fi
+
+if [ -d "/var/log/pods" ]
+then
+    sudo chgrp -R otel /var/log/pods
+    sudo chmod -R g+rwx /var/log/pods
+    sudo setfacl -Rm d:g:otel:rwx,g:otel:rwx /var/log/pods
+fi
+```
+
+
+## Architecture
```

Comment:
this pertains to collecting logs. so maybe call it "Log Architecture" or something more fitting. 
i am thinking it is better to be added as a separate doc file?

Reply:
updated - i think it fits here rather than a different section.